x-pack/plugins/fleet/common/constants/routes.ts

```diff
@@ -119,6 +120,7 @@ export const FLEET_PROXY_API_ROUTES = {
 export const SETTINGS_API_ROUTES = {
   INFO_PATTERN: `${API_ROOT}/settings`,
   UPDATE_PATTERN: `${API_ROOT}/settings`,
+  ENROLLMENT_INFO_PATTERN: `${API_ROOT}/settings/enrollment`,
```

Comment:
should we start with an internal route? does that API make sense for user or it's only valuable for our UI?

Reply:
I actually flagged it in the openapi spec as an internal one. I will change this to an internal route bc I don't think it should be used for end user consumption yet.

Reply:
I've moved this to an internal route. the openapi spec was updated too (pretty cool that it lets me override the global `/api/fleet` base path/server)